tests/e2e/utils/fixtures/base.ts

```diff
@@ -0,0 +1,23 @@
+import { mergeTests, test as base } from "@playwright/test";
+import { test as sarTest } from "./sar.ts";
+import { test as wpTest } from "./wp.ts";
+import StateHomePage from "../pageObjects/stateHome.page";
+import AdminHomePage from "../pageObjects/adminHome.page";
+
+type CustomFixtures = {
+  stateHomePage: StateHomePage;
+  adminHomePage: AdminHomePage;
+};
+
+export const baseTest = base.extend<CustomFixtures>({
+  stateHomePage: async ({ page }, use) => {
+    await use(new StateHomePage(page));
+  },
+  adminHomePage: async ({ page }, use) => {
+    await use(new AdminHomePage(page));
+  },
+});
+
+export const test = mergeTests(baseTest, sarTest, wpTest);
+
+export { expect } from "@playwright/test";
```

Comment:
I think because maybe it's cleaner this way? So that in our test files, we don't have to import `test` from our base fixture (where we are extending Playwright's `test`) and `expect` from Playwright separately. That's my best guess. It was done this way in the [docs](https://playwright.dev/docs/test-fixtures#creating-a-fixture), so I just copied.